foreman/start-manual.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+mdbook serve \
+  --port 63332 \
+  --dest-dir $(pwd)/.hydra-data/manual \
+  $(pwd)/doc/manual/
```

Comment:
```suggestion
  --dest-dir .hydra-data/manual \
  doc/manual/
```

Is there any advantage of adding pwd explicitly?